src/TLuaInterpreter.cpp

```diff
@@ -11123,26 +11123,26 @@ void TLuaInterpreter::msdp2Lua(char* src, int srclen)
 
             case MSDP_VAL:
                 if (last == MSDP_VAR) {
-                    script.append(R"(":)");
+                    script.append(QLatin1String(R"(":)"));
                 }
                 if (last == MSDP_VAL) {
                     no_array_marker_bug = true;
-                    script.append(R"(")");
+                    script.append(QLatin1Char('"'));
                 }
                 if (last == MSDP_VAL || last == MSDP_TABLE_CLOSE || last == MSDP_ARRAY_CLOSE) {
-                    script.append(",");
+                    script.append(QLatin1Char(','));
                 }
                 if (src[i + 1] != MSDP_TABLE_OPEN && src[i + 1] != MSDP_ARRAY_OPEN) {
-                    script.append(R"(")");
+                    script.append(QLatin1Char('"'));
                 }
                 varList.append(lastVar);
                 last = MSDP_VAL;
                 break;
             case '\\':
-                script.append(R"(\\)");
+                script.append(QLatin1String(R"(\\)"));
                 break;
             case '"':
-                script.append(R"(\")");
+                script.append(QLatin1String(R"(\")"));
```

Comment:
Similarly, __is__ `script.append(R"(\")");` correct - or doesn't the `R"(`...`")` wrapping also negate the need to escape the `\` here as well... 😕